<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config:SMi</class>
 <widget class="QDialog" name="Config:SMi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>221</width>
     <height>39</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="copyConfigButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy to selected SMi's</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="frame_FileSync">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>FileSync</string>
   </property>
   <widget class="QGroupBox" name="frame_FilePull">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Pull data from SMi</string>
    </property>
    <widget class="QToolButton" name="pullDirButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>24</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="pullDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;Destination Folder&gt;</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="pullButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="frame_FilePush">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Push data to SMi</string>
    </property>
    <widget class="QToolButton" name="pushDirButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>19</y>
       <width>24</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="pushDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;Source folder&gt;</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="frame_Audio">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>141</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Audio</string>
   </property>
   <widget class="QSlider" name="gainFader">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>16</width>
      <height>291</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="meters">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>System Health</string>
   </property>
   <widget class="QFrame" name="statemeter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="frame_Mode" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="frame_StreamSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Stream Settings</string>
    </property>
    <widget class="QComboBox" name="streamProtocol">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>RTP</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_streamProtocol">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>52</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_streamProfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>52</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_streamChannels">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>52</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Channels</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="streamChannels">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
    </widget>
    <widget class="QComboBox" name="streamProfile">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>L16</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QComboBox" name="modeSelector">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Streaming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recording</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Processing</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_mode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>52</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="frame_Network">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Network</string>
   </property>
   <widget class="QComboBox" name="networkInterface">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>eth0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wlan0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_networkInterface">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>52</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Interface</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Config:SMi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Config:SMi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
